benchmark/src/main/resources/application-borer-json.conf

```diff
@@ -0,0 +1,9 @@
+akka.actor {
+  serializers {
+    borer-json = "benchmark.borer.TestBorerJsonAkkaSerializer"
+  }
+  serialization-bindings {
+    "benchmark.model.MySerializable" = borer-json
+  }
+  enable-additional-serialization-bindings = on
```

Comment:
Nit: any way to extract this shared setting to `application.conf` or another shared config file?